contributors/devel/README.md

```diff
@@ -43,6 +43,9 @@ Guide](http://kubernetes.io/docs/admin/).
 * **Document Conventions** ([how-to-doc.md](how-to-doc.md))
   Document style advice for contributors.
 
+* **Conformance Tests Guidelines** ([conformance-guidelines.md](conformance-guidelines.md))
```

Comment:
I would argue this doc shouldn't just cover conformance test documentation.  We're also trying to describe what a conformance test is, what the process is to promote a test to conformance,  maybe even how to write a good one vs. a bad one, etc. Maybe this should be retitled and moved up closer to the "Testing" doc above?

Reply:
Fixed. Thanks for the review, please suggest if need be reworded,etc.